services/example01/webpack.config.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const slsw = require('serverless-webpack');
+const nodeExternals = require('webpack-node-externals');
+const path = require('path');
+
+module.exports = {
+  entry: slsw.lib.entries,
```

Comment:
I checked the documentation for the `serverless-webpack` but I didn't manage to understand what would be the workflow if there is an additional library which is to be developed separately from the main handler? For example, 1 file in `lib` contains an export of a single function which is imported in the handler, would the lib file be ok with the babel run-time transpiration if the handler is not called?

Reply:
Serverless has been updated during the weekend, I don't know which documentation you read but:
- AFAIK the code is not transpiled during runtime, it is transpiled before being deployed
- you can `import myFunction from 'helper.js'` and if there's no call to `myFunction`, the code won't be imported (tree-shaking)
But maybe I've missed your point